docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -163,3 +164,7 @@ To destroy the component:
 ```bash
 lokoctl component delete prometheus-operator --delete-namespace
 ```
+
+## Next steps
```

Comment:
I don't think that a "next steps" sections belongs in a config reference. This is not a tutorial. Also, we don't seem to have the "next steps" section in any of the other config reference docs.

Reply:
What is the best way to point users to this from the config reference?

Reply:
Maybe we can have an index page for all guides instead? I wouldn't put next steps section here.

Reply:
We already point to the howtos in the README, maybe that's enough for now?

Reply:
>What is the best way to point users to this from the config reference?

Personally I wouldn't point to a howto guide from a config reference guide. I might do the *opposite* though - point from the howto guide to the config reference.
Think about the "journey" the user will likely do: read a quickstart to get a cluster working, then read a howto guide to figure out how to set up monitoring, and lastly check the config reference to figure out how to customize something.